presentation/src/main/java/com/cointrend/presentation/ui/about/AboutScreen.kt

```diff
@@ -0,0 +1,236 @@
+                item {
+                    SectionTitle(title = "Contact", modifier = Modifier.padding(defaultHorizontalPadding))
+                }
+                item {
+                    Column(
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .background(
+                                color = StocksDarkBackgroundTranslucent,
+                                shape = MaterialTheme.shapes.large
+                            ),
+                    ) {
+                        SectionInfoItemTest(name = "Email", info = "cointrend.info@gmail.com", image = R.drawable.ic_gmail, showDivider = true)
+                        SectionInfoItemTest(name = "Github Issues", info = "https://github.com/CoinTrend/CoinTrend/issues", image = R.drawable.ic_github, showDivider = false)
+                    }
+                }
+                item {
+                    Spacer(modifier = Modifier.size(16.dp))
+                }
+                // Setion title 16dp SUPPORT
+                item {
+                    SectionTitle(title = "Support", modifier = Modifier.padding(defaultHorizontalPadding))
+                }
+                item {
+                    Column(
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .background(
+                                color = StocksDarkBackgroundTranslucent,
+                                shape = MaterialTheme.shapes.large
+                            ),
+                    ) {
+                        SectionInfoItemTest(name = "Donate Bitcoin", info = "https://github.com/CoinTrend/CoinTrend#bitcoin", image = R.drawable.ic_donate, showDivider = true)
+                        SectionInfoItemTest(name = "Rate on Google Play", info = "", image = R.drawable.ic_google_play, showDivider = false)
+                    }
+                }
+                item {
+                    Spacer(modifier = Modifier.size(32.dp))
+                }
+            }
+        }
+    }
+}
+
+
+//Logo CoinTrend Like Github Page
+@Composable
+private fun CoinTrend(
+    modifier: Modifier = Modifier
+
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+
+        ) {
+        Image(
+            modifier = Modifier
+                .requiredHeight(120.dp)
+                .padding(top = 2.dp, start = 115.dp),
+            contentScale = ContentScale.Fit,
+            painter = painterResource(id = R.drawable.ic_launcher),
+            contentDescription = null,
+        )
+        Text(
+            modifier = Modifier.padding(top = 2.dp, start = 115.dp),
+            text = "CoinTrend",
+            fontSize = 38.sp,
+            color = MaterialTheme.colorScheme.onBackground,
+            style = MaterialTheme.typography.bodySmall,
```

Comment:
Here try other styles which have a larger text size than the 'MaterialTheme.typography.titleLarge' one